sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/client/CosmosDb.java

```diff
@@ -62,7 +62,10 @@ private static String nowAsRFC1123() {
     }
 
     private static String urlEncode(String url) {
-        return urlEncode(url, "UTF-8");
+        if(url != null) {
+            return urlEncode(url, "UTF-8");
+        }
+        return "null";
```

Comment:
What is that return value? What do we do with it? It looks to me we need to return a document error instead if we get an error for url encoding.

Reply:
Ok that's because we mock token without fields that should be required such as db name or collection name and then url encoding on it. Added a TODO as we should have a bug to fix the root cause.